src/users/entities/user.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { AbstractEntity } from '../../utils';
+import { Authentication } from '../../auth/entities';
+import { Entity, Column, OneToOne, JoinColumn } from 'typeorm';
+
+@Entity({ name: 'users' })
+export class User extends AbstractEntity {
+  @Column({ unique: true })
+  public login: string;
+
+  @Column()
+  public firstName: string;
+
+  @Column()
+  public lastName: string;
+
+  @Column()
+  public address: string;
+
+  @Column()
+  public birthday: Date;
+
+  @Column({ default: false })
+  public isActive: boolean;
+
+  @OneToOne(() => Authentication, (auth: Authentication) => auth.user, { nullable: false, onDelete: 'CASCADE' })
+  @JoinColumn()
+  public authentication: Authentication;
```

Comment:
I think that you can remove this table, due to the fact that you can have password column in the user table